languages/english.php

```diff
@@ -116,3 +116,7 @@
 $lang["WITHPATH"] = "With Path:";
 $lang["FILEDELETED"] = "File deleted";
 $lang["FILEADDED"] = "New file";
+
+$lang['SEARCH'] = "Search";
+$lang['SEARCH_PLACEHOLDER'] = "Search for file name here";
+$lang['NOSEARCHTERM'] = "No Search Found";
```

Comment:
The key does not correspond to the message. It that supposed to be "No Items Found"?

Reply:
Yes it is No Items Found from the search term that was entered.

Reply:
The please align key and value.

Reply:
Changed the key to "NOITEMSFOUND" and value as "No Items Found". If this is okay will do the same for all languages.